walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,92 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 
 public final class FindMeetingQuery {
+
+  /**
+   * > 0 : false
+   * = 0 : true
+   */
+  private List<Integer> isAvailableAtMinute;
+
+  /**
+   * Worst Case
+   * Time Complexity: O(3 * 24 * 60 + events.size() * attendees.size())
```

Comment:
I think this is anyways O(1), so should we remove this?

Reply:
Yep.